README.md

```diff
@@ -75,6 +77,14 @@ CPU cores, you'll probably want to be compiling about 8 files simultaneously:
     make -j8
 ```
 
+If you wish to run the test suite, make sure you have a database set up so that
+you will connect and log in automatically, without a password prompt.  See the
+section on variables such as `PGHOST` and `PGUSER`.  **The tests will create
+and drop tables in that database,** so don't use a database containing any
+valuable data.
+
+To run the tests, run both `test/runner` and `test/unit/unit_runner`.
+
```

Comment:
You might consider moving the documentation changes that aren't pertinent to the streaming code to their own PR, to simplify review of the streaming code.

Reply:
Well I was hoping to merge all this in much shorter order, but life intervened.